src/Plugins/SimplnxCore/test/CreateAMScanPathsTest.cpp

```diff
@@ -63,37 +63,38 @@ TEST_CASE("SimplnxCore::CreateAMScanPathsFilter: Valid Filter Execution", "[Simp
   auto result = filter.execute(dataStructure, args);
   SIMPLNX_RESULT_REQUIRE_VALID(result.result)
 
-  // Compare the exemplar and the computed outputs
-  {
-    auto exemplarGeom = dataStructure.getDataAs<IGeometry>(k_ExemplarScanVectorsPath);
-    auto computedGeom = dataStructure.getDataAs<IGeometry>(k_ComputedScanVectorsPath);
-    REQUIRE(UnitTest::CompareIGeometry(exemplarGeom, computedGeom));
-  }
-  {
-    DataPath exemplarDataArray = k_ExemplarScanVectorsPath.createChildPath(k_VertexData).createChildPath(k_Times);
-    DataPath computedDataArray = k_ComputedScanVectorsPath.createChildPath(k_VertexData).createChildPath(k_Times);
-    UnitTest::CompareArrays<float64>(dataStructure, exemplarDataArray, computedDataArray);
-  }
-
-  {
-    DataPath exemplarDataArray = k_ExemplarScanVectorsPath.createChildPath(k_EdgeData).createChildPath(k_RegionIdsName);
-    DataPath computedDataArray = k_ComputedScanVectorsPath.createChildPath(k_EdgeData).createChildPath(k_RegionIdsName);
-    UnitTest::CompareArrays<int32>(dataStructure, exemplarDataArray, computedDataArray);
-  }
-  {
-    DataPath exemplarDataArray = k_ExemplarScanVectorsPath.createChildPath(k_EdgeData).createChildPath(k_Powers);
-    DataPath computedDataArray = k_ComputedScanVectorsPath.createChildPath(k_EdgeData).createChildPath(k_Powers);
-    UnitTest::CompareArrays<float32>(dataStructure, exemplarDataArray, computedDataArray);
-  }
-
-  {
-    DataPath exemplarDataArray = k_ExemplarScanVectorsPath.createChildPath(k_EdgeData).createChildPath(k_SliceIdsPath.getTargetName());
-    DataPath computedDataArray = k_ComputedScanVectorsPath.createChildPath(k_EdgeData).createChildPath(k_SliceIdsPath.getTargetName());
-    UnitTest::CompareArrays<int32>(dataStructure, exemplarDataArray, computedDataArray);
-  }
+// Write out the .dream3d file now
+#ifndef SIMPLNX_WRITE_TEST_OUTPUT
+  std::cout << "Writing File: " << fmt::format("{}/create_am_scan_paths_test.dream3d", unit_test::k_BinaryTestOutputDir) << "\n";
+  WriteTestDataStructure(dataStructure, fmt::format("{}/create_am_scan_paths_test.dream3d", unit_test::k_BinaryTestOutputDir));
+#endif
+
+  //  // Compare the exemplar and the computed outputs
```

Comment:
Are these comments here for debugging or artifacts of test updates? If the former put a comment mentioning the use case for future updaters

Reply:
Well crap. I should have enabled some of these comparisons. I will get a new commit pushed up.